plans/friends/beakerlib.fmf

```diff
@@ -0,0 +1,15 @@
+summary: Integration plan with beakerlib
+
+description:
+    Ensure that the basic functionality of tests written in
+    beakerlib is working as expected. This plan is run for
+    upstream pull requests in the beakerlib repository.
+
+discover:
+    how: fmf
+    url: https://github.com/teemtee/tmt
+    ref: tag-beakerlib-tests
+    filter: tag:beakerlib
```

Comment:
Indeed that is what I had in mind with a reverse-integration test. This can be simplified though as:
```yaml
discover:
  how: fmf
  path: /
```

I have something similar with multi-fmf trees in [spglib](https://github.com/spglib/spglib/blob/develop/.distro/plans/full.fmf).

The only thing I am not sure is how well it plays with installing both upstream and downstream package. Probably including a `prepare.install.tmt` (+`yq`, etc.) is sufficient

Reply:
I believe, providing just `path: /` would not work. If `url` is not present, an absolute filesystem path is expected:

 * https://tmt.readthedocs.io/en/stable/spec/plans.html#fmf

Reply:
why special branch?

Reply:
On the spglib side it works as:
```yaml
discover:
  how: fmf
  path: test
```
There I have the second tree in `/test` folder, but it should work as `/` or `.`. The triggering plan is located in `.distro/plans/full.fmf` with the tree `.distro`

Reply:
aah, sorry, I need to play with ref for now (until PR is merged)

Reply:
When I implemented on `spglib` it worked without the `ref`, granted I ran it on the same repo. I think the issue is more with the `filter`. Other than probably using the `"` I didn't see any issues with the previous implementation though :thinking:

Reply:
Was there a separate `fmf` subtree in your case as well? Here we added `.fmf/version` in order to prevent running friendly plans for `tmt` itself.

Reply:
Indeed, but see #2737

Reply:
For now, the current approach works. And thinking about it again. It is weird that it works now without a `ref`, or maybe it was removed to prepare for the merge. Anyway, we did check that it will work if merged in the last round.